src/utils/internal.ts

```diff
@@ -1891,3 +1898,75 @@ export function areSameClaims(
 
   return ClaimType[type] === claim.type;
 }
+
+/**
+ * @hidden
+ */
+export function throwIfPendingAuthorizationExists(params: {
```

Comment:
```suggestion
export function assertNoPendingAuthorizationExists(params: {
```

I think we normally use `assert` prefix for these types of functions